android/app/build.gradle

```diff
@@ -194,8 +194,8 @@ android {
         applicationId "me.rainbow"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 171
-        versionName "1.0"
+        versionCode 172
```

Comment:
version code can be set back to 1, we won't be using this for now until we set up fastlane for android